rpm/buteo-sync-plugin-caldav.spec

```diff
@@ -76,14 +76,6 @@ This package contains unit tests for the CalDAV Buteo sync plugin.
 %qmake5
 make %{?_smp_mflags}
```

Comment:
Not related to the PR, but maybe one can change here the `make` statement with hardcoded parallel compilation, for an RPM macro. I think there is one for this, isn't there ? But I can't find it anymore :/

Reply:
%make_build. I'm somewhere on the borderline which is better, maybe not bother changing this time.